tests/unit/replicator_tests.py

```diff
@@ -210,6 +211,47 @@ def test_create_replication(self):
             ])
         )
 
+    def test_timeout_in_create_replication(self):
```

Comment:
`test_create_replication` was one of a few tests that failed consistently when `timeout=1` and so I went ahead with creating this unit test.   The tests are even more unstable with `timeout=.1`.